gradle/dependency-versions.gradle

```diff
@@ -19,8 +19,8 @@ ext {
     bouncycastleVersion = "1.64"
     javaxVersion = "3.0.1"
     nettyVersion = "4.1.79.Final"
-    nettyTcNativeBoringSSLStatic_linux_x86_64 = "2.0.54.Final:linux-x86_64"
-    nettyTcNativeBoringSSLStatic_osx_aarch_64 = "2.0.54.Final:osx-aarch_64"
+    nettyTcNativeBoringSSLStatic_linux_x86_64 = "2.0.55.Final:linux-x86_64"
+    nettyTcNativeBoringSSLStatic_osx_aarch_64 = "2.0.55.Final:osx-aarch_64"
```

Comment:
- Dependency has been downgraded, please upgrade the following dependencies. - (WARN) io.netty:netty-tcnative-boringssl-static was pulled up transitively to 2.0.61.Final but the product-spec is forcing the version to 2.0.54.Final

Can we do 2.0.61 ?

Reply:
done